src/pages/patterns/search-pattern/index.mdx

```diff
@@ -0,0 +1,419 @@
+            results page
+          </StructuredListCell>
+          <StructuredListCell>
+            When a user will conduct a search and be routed to a distinct
+            results page.
+          </StructuredListCell>
+        </StructuredListRow>
+        <StructuredListRow>
+          <StructuredListCell>
+            <a href="#active-search">Active search</a>
+          </StructuredListCell>
+          <StructuredListCell>
+            Actively searches a database and lists the top results in results
+            panel or on the current page. For example:
+            <UnorderedList>
+              <ListItem>Searching a catalog</ListItem>
+              <ListItem>Searching a small website</ListItem>
+              <ListItem>Searching a small data set within a table</ListItem>
+              <ListItem>
+                Searching a subset of information embedded within a product or
+                application's page
+              </ListItem>
+            </UnorderedList>
+          </StructuredListCell>
+          <StructuredListCell>
+            Best for small data sets, like a single page, website, or table.
+            When a user will benefit from constant feedback based on their
+            search query, and when the server can handle substantial search
+            query load.
+          </StructuredListCell>
+        </StructuredListRow>
+        <StructuredListRow>
+          <StructuredListCell>
+            <a href="#focused-search">Focused search</a>
+          </StructuredListCell>
+          <StructuredListCell>
+            <UnorderedList>
+              <ListItem>Assigned tasks</ListItem>
+              <ListItem>Product catalogs</ListItem>
+              <ListItem>Personal repositories</ListItem>
+              <ListItem>Work created by the active user</ListItem>
+            </UnorderedList>
+          </StructuredListCell>
+          <StructuredListCell>
+            When the search is focused on tasks or information specific to the
+            active user. Limits server load by focusing search scope while
+            offering the power of a broad, basic search.
+          </StructuredListCell>
+        </StructuredListRow>
+      </StructuredListBody>
+    </StructuredListWrapper>
+  </Column>
+</Row>
+
+### Best practices
+
+#### Avoid dead ends
+
+If a search returns "No results," suggest a follow-up action. Provide suggestions and helpful resources to ensure the user finds what they are looking for. For more on failed searches, see the [empty states pattern](/patterns/empty-states-pattern).
+
+#### Include a loading indicator
+
+Include a loading indicator if the search will take longer than a moment or two. Your loading state should reflect your empty state with useful helper text signaling that the search is still ongoing. Include a progress bar for advanced, resource-intensive searches. Let the user know how far along the search is, and roughly how long it will take.
```

Comment:
```suggestion
If the search will take longer than a moment or two, include a loading indicator. Your loading state should reflect your empty state with useful helper text signaling that the search is still ongoing. Include a progress bar for advanced, resource-intensive searches. Let the user know how far along the search is, and roughly how long it will take to finish.
```